connection/cluster/ClusterDatabaseManager.java

```diff
@@ -59,25 +59,58 @@ public ClusterDatabaseManager(ClusterClient client) {
 
     @Override
     public boolean contains(String name) {
-        return failsafeTask(name, (stub, dbMgr) -> dbMgr.contains(name));
+        try {
+            BiFunction<ClusterServerStub, TypeDBDatabaseManagerImpl, Boolean> task = ((stub, dbMgr) -> dbMgr.contains(name));
+            return client.createFailsafeTask(
+                    name,
+                    parameter -> task.apply(
+                            parameter.client().stub(),
+                            parameter.client().databases()
+                    )
+            ).runPrimaryReplica();
+        } catch (TypeDBClientException e) {
+            if (e.getErrorMessage().equals(DB_DOES_NOT_EXIST)) return false;
+            else throw e;
+        }
     }
 
     @Override
     public void create(String name) {
-        failsafeTask(name, (stub, dbMgr) -> {
+        BiFunction<ClusterServerStub, TypeDBDatabaseManagerImpl, Void> task = ((stub, dbMgr) -> {
             dbMgr.create(name);
             return null;
         });
+        ClusterClient.FailsafeTask<Void> failsafeTask = client.createFailsafeTask(
+                name,
+                parameter -> task.apply(
+                        parameter.client().stub(),
+                        parameter.client().databases()
+                )
+        );
+        try {
+            failsafeTask.runAnyReplica();
+        } catch (TypeDBClientException e) {
+            if (CLUSTER_REPLICA_NOT_PRIMARY.equals(e.getErrorMessage())) {
+                failsafeTask.runPrimaryReplica();
+            } else throw e;
+        }
```

Comment:
**If the database never existed**, there is no primary, so we must run it on any replica. This may fail in two ways:
1. While creating replicators on each node: 
2. When writing the database `CREATE` management record to the raft log. 
* If this is caused by the (highly probable) scenario where the node the client is connected not being voted primary (for the newly created replica), we transparently retry on whichever node is the primary.

In all other cases, We require the user to retry. This is safe because database creation is idempotent.

If the database did exist, we should be running create on the primary.
The runAnyReplica call will fail with `CLUSTER_REPLICA_NOT_PRIMARY` and retry on the primary.